Release-Checklist.md

```diff
@@ -0,0 +1,79 @@
+Change release process, instead create a release candidate we can test the HEAD what will be the last commit using new automated obs package generation we can generate and test commits based on HEAD.
```

Comment:
This sentance doesn't read well - can we clean that up a bit?
Also, can we have a #TITLE here at the top as well (right now it just dives right on in..)

Reply:
Some point to make clear are:
- By sending a PR to update the VERSION file, we exercise the CI that tell us the project health.
- OBS packages are created after we do tag the project repositories based in the tag we create.

Reply:
Oh - that shouldn't even be there - zapped ;)